site/content/documentation/tutorials/sparqlbuilder.md

```diff
@@ -150,53 +153,53 @@ GraphPatternNotTriple union = GraphPatterns.union(book.has(dc10.iri("title"), ti
 	book.has(dc11.iri("title"), title);
 System.out.println(union.getQueryString());
 // ==> { ?book dc10:title ?title } UNION { ?book dc11:title ?title }
-{{< / highlight >}}
+```
 
 `GraphPatterns#optional()` creates an optional group graph pattern, consisting of the passed in `GraphPattern`s:
 
-{{< highlight java "linenos=table" >}}
+```java
 GraphPatternNotTriple optionalPattern = GraphPatterns.optional(GraphPatterns.tp(x, foaf.iri("mbox"), mbox));
 System.out.println(optionalPattern.getQueryString());
 // ==> OPTIONAL { ?x foaf:mbox ?mbox }
-{{< / highlight >}}
+```
 
 ### Sub-select
 
 Finally, GraphPatterns#select() creates an instance of a SubSelect, which represents a SPARQL subquery:
 
-{{< highlight java "linenos=table" >}}
+```java
 SubSelect subQuery = GraphPatterns.select();
-{{< / highlight >}}
+```
 
 ## Query Constraints
 
 You can create SPARQL query constraints using the Expressions class which provides static methods to create Expression objects representing SPARQL’s built-in expressions:
 
-{{< highlight java "linenos=table" >}}
+```java
 Variable name = SparqlBuilder.var("name");
 Expression<?> regexExpression = Expressions.regex(name, "Smith");
 System.out.println(regexExpression.getQueryString());
 // ==> REGEX( ?name, "Smith" )
-{{< / highlight >}}
+```
 
 `Expression`s take `Operand` instances as arguments. Operand is implemented by the types you would expect (Variable, the RDF model interface RdfValue, and Expression itself). Where possible, Expressions has included wrappers to take appropriate Java primitives as parameters as well:
 
-{{< highlight java "linenos=table" >}}
+```java
 Variable price = SparqlBuilder.var("price");
 Expression<?> priceLimit = Expressions.lt(price, 100);
 System.out.println(priceLimit.getQueryString());
 // ==> ?price < 100
-{{< / highlight >}}
+```
 
 For those places where a wrapper has not (yet) been created, `RdfLiteral`s can be used instead. The `Rdf` class provides factory methods to create StringLiteral, NumericLiteral, and BooleanLiteral instances:
 
-{{< highlight java "linenos=table" >}}
+```java
 Variable price = SparqlBuilder.var("price");
 ExpressionOperand discount = Rdf.literalOf(0.9);
 Expression<?> discountedPrice = Expressions.multiply(price, discount);
 System.out.println(discountedPrice.getQueryString());
 // ==> ( ?price * 0.9 )
-{{< / highlight >}}
+```
```

Comment:
So java highlighting works now?

Reply:
Yup! It has for a while actually, we just didn't update all pages yet.